CONTRIBUTING.md

```diff
@@ -624,32 +624,36 @@ We use [Cypress](https://www.cypress.io/) for integration tests. Tests can be ru
 
 ```bash
 export SUPERSET_CONFIG=tests.superset_test_config
+export SUPERSET_TESTENV=true
+export ENABLE_REACT_CRUD_VIEWS=true
 export CYPRESS_BASE_URL="http://localhost:8081"
 superset db upgrade
-superset init
 superset load_test_users
-superset load_examples
+superset load_examples --load-test-data
```

Comment:
👍

Reply:
Do we want to drop database before? It can be done with modification of cli.py
```@superset.command()
@with_appcontext
def create_db() -> None:
    """Resets database"""
    db.create_all()


@superset.command()
@with_appcontext
@click.confirmation_option(prompt="Are you sure you want to drop the db?")
def drop_db() -> None:
    """Removes database"""
    db.drop_all()
```
And then executed: 
```
superset drop_db --yes
superset create_db
```

Reply:
@adam-stasiak definitely see the value in having a drop db (or at least a drop table) function prior to running tests, but I wouldn't necessarily make it mandatory.